src/logistration/LoginHelpLinks.jsx

```diff
@@ -41,7 +42,7 @@ const LoginHelpLinks = (props) => {
   const renderLoginHelp = () => (
     <div className="login-help">
       { page === LOGIN_PAGE ? forgotPasswordLink() : signUpLink()}
-      <a className="field-link" href="https://support.edx.org/hc/en-us/sections/115004153367-Solve-a-Sign-in-Problem">
+      <a className="field-link" href={getConfig().LOGIN_ISSUE_SUPPORT_LINK}>
```

Comment:
Wrap this inside an if clause, if this variable is defined then render the link.